airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2110,6 +2110,16 @@ components:
           format: uri
         icon:
           type: string
+        releaseStage:
```

Comment:
Could re-use the release stage across both of these (see `ConnectionStatus` for an example of a re-used string enum).